server/app.js

```diff
@@ -1,21 +1,29 @@
-const express = require("express");
-const morgan = require("morgan");
-const cors = require("cors");
-const bodyParser = require("body-parser");
-const mongoose = require("mongoose");
+const express = require("express")
+const morgan = require("morgan")
+const cors = require("cors")
+const bodyParser = require("body-parser")
+const app = express()
+ 
+//parsers
+app.use(cors())
+app.use(morgan("dev"))
+app.use(bodyParser.urlencoded({ extended: false }))
+app.use(bodyParser.json())
 
-const app = express();
+// server client files
+app.use(express.static(__dirname + "/../client/build"))
```

Comment:
We don't need this.

Reply:
how wil we serve the client side file

Reply:
Both client and server will be served independently and client will talk with server using APIs.